frontend/src/static/css/style.scss

```diff
@@ -685,8 +685,30 @@ header {
     }
   }
 
-  &__toggle-commit-message-body {
-    padding-top: 10px;
+  &__toolbar {
+    align-items: center;
+    display: flex;
+
+    .sorting {
+      margin-left: auto;
+
+      .mui-select {
+        cursor: pointer;
+        padding-right: 15px;
+      }
+
+      label {
+        overflow-y: hidden;
+      }
+
+      & > label {
+        margin-left: 10px;
+      }
+    }
+
+    .toggle-commit-message-body {
+      width: 50%;
+    }
```

Comment:
Seems overkill given that we're reusing existing style. 
Can't we just standardize the both of them to be on the left side and save all this changes?
The `hide all commit messages body` can be changed to how `Collapse all` is like in authorship tab

Reply:
For the `hide all commit messages body`, there is already a issue regarding this (#993 ). Can I check if you prefer tick box or the authorship style? If the authorship style, you want me to make the change in this PR?

Reply:
Authorship style would be better, since its also a toggle button.
Can add it to this PR if its light changes.